src/fdosecrets/widgets/SettingsWidgetFdoSecrets.ui

```diff
@@ -46,21 +46,14 @@
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
-       <item>
-        <widget class="QCheckBox" name="noConfirmDeleteItem">
-         <property name="toolTip">
-          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If enabled, clients can delete exposed entries without confirmation. Otherwise, attempts to delete an entry may need to be confirmed interactively (subject to global Security/Convenience settings).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
I suggest keeping the tooltip. This option does not ensure the user is always prompted when entries are deleted due to the new Security setting.

Reply:
Sure I will reword it though to be simple, something like "This setting does not override disabling recycle bin prompts"